### ISSUE TITLE
Ensure lists for indentation are not used

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#65]
## Description
The HTML list elements `dl`, `ul`, and `ol` should only be used to create lists, not for formatting effects such as indentation.

### Breadcrumb
- [ ] Breadcrumb links should only be organized in a list if there is more than 1 item in the list

<img width="940" alt="9837a65c2dfd430dd2d811d9b427d0753351a072-1641278333" src="https://user-images.githubusercontent.com/8166060/153066994-d395bce0-84ee-4519-bd2f-689f8ab399e7.png">

## CIC Estimate
0 points

**Comment by dananjohnson** [comment]
@blnkt the reason for the list element here is that this is a generic breadcrumbs component that will render the array of links it gets into a list. It doesn't really consider how many links are present. I replaced the markup with a more accessible alternative in https://github.com/lsst-epo/rubin-obs-client/commit/cc9c0b7c941199a029807ed9ae5df38ec6214dd2, but that doesn't really address the specific issue raised here. I hadn't considered before that we should omit the `ol > li` structure if the link array is only one item long. I wonder if it would be better just to hide the breadcrumb entirely if there's only one item? It doesn't seem very meaningful otherwise. In the particular screenshot above, it seems like there should be a second item, like "Staff". What do you think?

**Comment by blnkt** [comment]
You're right, in the above screenshot it does feel like something's missing......but dismissing that for the time being, I agree: a breadcrumb only one item long does not feel particularly meaningful.  Let's omit the breadcrumb altogether if it is only 1 item long.
